src/main/kotlin/org/digma/intellij/plugin/ui/list/insights/SessionInViewSpansPanel.kt

```diff
@@ -125,6 +126,7 @@ private fun addMainDescriptionLabelWithLink(spanOneRecordPanel: JPanel, displayT
     if ( StringUtils.isNotEmpty(displayText)) {
         val normalizedDisplayName = StringUtils.normalizeSpace(displayText)
         val actionLink = ActionLink(normalizedDisplayName) {
+            ActivityMonitor.getInstance(project).registerSpanLinkClicked("Main", InsightType.EndpointSessionInView.toString())
```

Comment:
maybe instead `Main` lets call it `Insights` (using `PanelName` enum)

Reply:
added